src/utils/createI13nNode.js

```diff
@@ -31,6 +31,16 @@ function objectWithoutProperties(obj, keys) {
     return target;
 }
 
+function mixin(target, source) {
```

Comment:
what I do for this is basically extend React.Component to get a basic component class, and do prototype updating here. 

I'm not sure if we have better way to introduce new functionalities, if we do please let me know.